TCMSApp/src/client/app/tests/tests-list.html

```diff
@@ -0,0 +1,172 @@
+                  <th>Assignee</th>
+                  <th>Run</th>
+                </tr>
+              </thead>
+
+              <tbody ng-repeat="tc in vmTests.currentSuite.tests" style="border:0">
+                <tr ng-click="seeCase = !seeCase;" style="cursor: pointer">
+                  <td ng-click="seeCase = ! seeCase">
+                    <input type="checkbox" ng-checked="vmTests.checkedAllCases">
+                  </td>
+                  <td>{{ ::tc.testName }}</td>
+                  <td>{{ ::vmTests.currentSuite._id + "-" + tc._id}}</td>
+                  <td>{{ ::tc.casePriority }}</td>
+                  <td>{{ ::tc.dateStart | date }}</td>
+                  <td>{{ ::tc.automated }}</td>
+                  <td>{{ ::tc.steps.length }}</td>
+                  <td>{{ ::tc.issues }}</td>
+                  <td>{{ ::tc.assignee }}</td>
+                  <td><a ui-sref="run-execute"><i class="fa fa-play-circle fa-lg"></i></a></td>
+                </tr>
+                <tr ng-if="seeCase" style="background-color: #eee">
+                  <td colspan="10">
+                    <div class="container-fluid text-center" style="padding: 20px 0">
+                      <div class="clearfix">
+                        <div class="col-xs-2"><b>Sprint: </b>{{ ::tc.sprint }}</div>
+                        <div class="col-xs-3"><b>Category: </b>{{ ::tc.category }}</div>
+                        <div class="col-xs-4"><b>Reference: </b>{{ ::tc.reference }}</div>
+                        <div class="col-xs-3"><b>File: </b><a href="#">{{ ::tc.fileName }}.zip</i></a></div>
+                      </div>
+                    </div>
+                    <div class="container-fluid" style="padding-bottom: 20px">
+                      <div class="clearfix">
+                        <div class="col-xs-12">{{ ::tc.preConditions }}</div>
+                      </div>
+                    </div>
+                    <table class="table">
+                      <thead>
+                        <tr>
+                          <th>Step</th>
+                          <th>Step info</th>
+                          <th>Expected result</th>
+                          <th>File</th>
+                        </tr>
+                      </thead>
+                      <tbody>
+                        <tr ng-repeat="step in tc.steps">
+                          <td>{{ ::step.stepNumber }}</td>
+                          <td>{{ ::step.stepDescription }}</td>
+                          <td>{{ ::step.expectedResult }}</td>
+                          <td>download</td>
+                        </tr>
+                      </tbody>
+                    </table>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div> <!-- panel -->
+
+    </div> <!-- row -->
+
+</div> <!-- container -->
+</div> <!-- matter -->
+
+
+<script type="text/ng-template" id="newSuite.html">
```

Comment:
without any pain we can move this template to the separate file named new-suite-modal.